### PR TITLE
Documentation updates on generators and symbols

### PR DESCRIPTION
**Pull request by vlada-shubina**
### Solution
Improves the documentation on generators and symbols

### Checks:
- [ ] Added unit tests - NA
- [ ] Added `#nullable enable` to all the modified files [?](https://github.com/dotnet/templating/wiki/Contributing#coding-style) - NA

### CODE REVIEW

File: docs/Available-Symbols-Generators.md [hunk 1]
```diff
@@ -182,42 +183,17 @@ Uses constant value.
...
 
 #### Parameters
 
-| Name     |Data Type| Description   |
-|----------|------|---------------|
-|fallback|string|fallback value| 
+|Name|Data Type|Description|Mandatory|
+|----------|------|---------------|---|
+|`high`|`integer`|defined the high bound of range to select port from|no, default: `1024`|
+|`low`|`integer`|defined the low bound of range to select port from|no, default: `65535`|
```
Comment (JanKrivanek) [inline docs/Available-Symbols-Generators.md:195]: high and low defaults switched here

Comment (JanKrivanek) [reply docs/Available-Symbols-Generators.md:195]: IIRC the bounds are forces by the code (e.g. if low is specified to be below 1024, it is silently adjusted to 1024) - if it's so, it's worth documenting. e.g. "The low bound of range for port selection. Will be adjusted to range <1024; 65535> if specified outside of this range."

File: docs/Available-Symbols-Generators.md [hunk 2]
```diff
@@ -305,10 +281,10 @@ This sample creates different symbols showing the different formatting available
 Creates a symbol from the current date/time. 
 
 #### Parameters 
-| Name     |Data Type| Description   |
-|----------|------|---------------|
-|format|string|DateTime.ToString format|
-|utc|bool|UTC time if true, local time otherwise|    
+|Name|Data Type|Description|Mandatory|
+|----------|------|---------------|---|
+|`format`|`string`|`DateTime.ToString()` format|no|
```
Comment (JanKrivanek) [inline docs/Available-Symbols-Generators.md:286]: Do we want to link msdne here? E.g. https://learn.microsoft.com/en-us/dotnet/standard/base-types/standard-date-and-time-format-strings

File: docs/Available-Symbols-Generators.md [hunk 3]
```diff
@@ -336,10 +312,10 @@ In this sample a symbol is created showing the current data, and replacing any i
...
 
 #### Parameters
-| Name     |Data Type| Description   |
-|----------|------|---------------|
-|low|integer|lower bound|
-|high|integer|upper bound|   
+|Name|Data Type|Description|Mandatory|
+|----------|------|---------------|---|
+|`low`|`integer`|lower bound|yes|
+|`high`|`integer`|upper bound|no, default:`int.MaxValue`|
```
Comment (JanKrivanek) [inline docs/Available-Symbols-Generators.md:318]: 'lower inclusive bound'
'upper exclusive bound'

**Review by JanKrivanek — APPROVED** [review]
Nice improvements!

I have couple of suggestions, but even without those this looks ready to go

File: docs/Available-Symbols-Generators.md (hunk 1 above)
Comment (vlada-shubina) [reply docs/Available-Symbols-Generators.md]: also added reserved ports

**Pull request merged**